python/requests/security/disabled-cert-validation.yaml

```diff
@@ -23,3 +23,6 @@ rules:
       - pattern: requests.request(..., verify=False, ...)
       - pattern: requests.get(..., verify=False, ...)
       - pattern: requests.post(..., verify=False, ...)
+    fix-regex:
+      regex: verify(\s)*=(\s)*False
```

Comment:
This is correct but developers will hate this - SSL verification is often disabled in test environments because provisioning machine keys for SSL requires some domain expertise & dev environment setup scripting.